index.html

```diff
@@ -53,5 +53,226 @@ <h3 class="job-title">
             <div id="background-img" class="background-img"></div>
         </div>
     </section>
+    <!-- Works section -->
+    <section>
+        <div class="works">
+            <div class="title">
+                <h2 class="recentWorks">
+                    My Recent Works
+                </h2>
+                <img src="images/line.png" alt="line">
+            </div>
+            <div class="work1">
+                <img src="images/Img Placeholder.png" alt="image">
+                <h3 class="work1Title">
+                    Multi-Post Stories
+                </h3>
+                <p class="work1Description">
+                    A daily selection of privately personalized reads; no accounts or 
+                    sign-ups required. has been the industry's standard dummy text ever 
+                    since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="work1Languages">
+                    <li> <button class="work1Button" type="button">CSS</button> </li>
+                    <li> <button class="work1Button" type="button">HTML </button> </li>
+                    <li> <button class="work1Button" type="button">Bootstrap</button> </li>
+                    <li> <button class="work1Button" type="button">Ruby</button> </li>
+                </ul>
+                <button class="work1Link">See project</button>
+            </div>
+            <div class="otherWorks">
+                <div class="work2">
+                    <h3 class="work2Title">
+                        Profesional Art Printing Data
+                    </h3>
+                    <p class="work2Description">
+                        A daily selection of privately personalized reads; no accounts or 
+                        sign-ups required. Has been the industry's standard.
+                    </p>
+                    <ul class="work2Languages">
+                        <li> <button class="work2Button" type="button">HTML </button> </li>
+                        <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+                        <li> <button class="work2Button" type="button">Ruby</button> </li>
+                    </ul>
+                    <button class="work1Link">See project</button>
+                </div>
+                <div class="work2">
+                    <h3 class="work2Title">
+                        Profesional Art Printing Data
+                    </h3>
```

Comment:
- [optional] great job with the button interactions. please kindly align your cards with the upper section like in the original design.
![Screenshot (902)_LI](https://user-images.githubusercontent.com/65298048/227720791-ffeee79a-4c6a-4331-ade9-b346f2b1f657.jpg)